scaleway/resource_ip.go

```diff
@@ -32,6 +32,8 @@ func resourceScalewayIP() *schema.Resource {
 			"reverse": {
 				Type:        schema.TypeString,
 				Optional:    true,
+				Computed:    true,
+				Deprecated:  "use scaleway_ip_reverse_dns resource instead",
```

Comment:
You should modify the docs for `scaleway_ip` as well.

Reply:
good point, I’ll add the deprecation notice there too.

Reply:
updated! @paultyng PTAL